playbooks/adhoc/uninstall.yml

```diff
@@ -329,21 +328,16 @@
       dest=/etc/sysconfig/docker
       regexp='(ADD_REGISTRY|BLOCK_REGISTRY|INSECURE_REGISTRY)=.*'
 
-  - name: Detect Docker storage configuration
-    shell: vgs -o name | grep docker
-    register: docker_vg_name
-    failed_when: false
-    changed_when: false
-
-  - name: Wipe out Docker storage contents
-    command: vgremove -f {{ item }}
-    with_items: "{{ docker_vg_name.stdout_lines }}"
-    when: docker_vg_name.rc == 0
+  - name: Umount docker storage if mounted
```

Comment:
This is not necessary.  `/usr/bin/docker-storage-setup` calls umount for  /var/lib/docker /var/lib/docker/containers /var/lib/docker/overlay2.